### ISSUE TITLE
UnixProcess getting killed

### ISSUE THREAD

**Issue by bennetth** [zeroturnaround/zt-exec#65]
I have a spring boot application that launches child processes using ProcessExecutor...start(). I'm expecting the child processes to continue running when I stop the spring boot application. When I run the application on LINUX the processes are killed when I stop the main application. However, when I stop the application on Windows the processes continue to run as expected. Is there a way to prevent the processes from stopping on LINUX?

**Comment by reinra** [comment]
Hey,
How do you stop the application on both LINUX and Windows?
On LINUX if you do Ctrl+C then it sends the termination signal not to the child process alone but to an entire "job" which means child process and grand child processes etc. As an alternative find the PID number via "ps" and then kill only that single process. Check if the grand child processes remain running in that case.

**Comment by bennetth** [comment]
@reinra: Thank you for your reply. So I've made some progress on this. I was using Ctrl+C during earlier testing (switching to systemd service) and so the child processes were getting killed as you mentioned. So I've updated the command being executed so that the process is separated from the main application (parent process). I used `setsid` to accomplish that and it's working as expected. This issue can be closed.
